.vscode/settings.json

```diff
@@ -33,5 +33,6 @@
         "Visio",
         "WEBP",
         "WebSocket"
-    ]
+    ],
+    "dotnet.defaultSolution": "disable"
```

Comment:
@jaklke, just to be sure, did you intend to make this change? At first sight, I don't really see the relevance for the rest of the PR.